README.md

```diff
@@ -10,15 +10,12 @@ Contact us at <support@chartiq.com> to request sample code and guidance on how t
 
 ## Requirements
 
-- Version 9.0.1 or later of the ChartIQ library
+- A copy of the ChartIQ JavaScript library (works best with version 9.0.1).
```

Comment:
```suggestion
- A copy of the ChartIQ JavaScript library (works best with version 9.0.1 or later).
```

Reply:
Per Gus: 
"we don't want to say “or later” because that may not be the case. The later version may have breaking changes, and if we have not made the changes on the wrappers, it won't work. 

I rather keep the “works best”"